app/components/SystemMetric.tsx

```diff
@@ -177,20 +175,18 @@ export function SystemMetric({
         {(inRange.isPending || beforeStart.isPending) && <Spinner />}
       </h2>
       {/* TODO: proper skeleton for empty chart */}
-      <Suspense fallback={<div />}>
-        <div className="mt-3 h-[300px]">
-          <TimeSeriesChart
-            data={data}
-            title={title}
-            width={480}
-            height={240}
-            interpolation="stepAfter"
-            startTime={startTime}
-            endTime={endTime}
-            unit={unit !== 'count' ? unit : undefined}
-          />
-        </div>
-      </Suspense>
+      <div className="mt-3 h-[300px]">
+        <TimeSeriesChart
+          data={data}
+          title={title}
+          width={480}
+          height={240}
+          interpolation="stepAfter"
+          startTime={startTime}
+          endTime={endTime}
+          unit={unit !== 'count' ? unit : undefined}
+        />
+      </div>
```

Comment:
I got rid of the Suspense here in 2f38adcd9d511a2c4895ca7ac0d752ee350e1344 because we're relying on RR's lazy loading instead of doing it manually.